backend/local/local.go

```diff
@@ -1423,6 +1427,8 @@ func cleanRootPath(s string, noUNC bool, enc encoder.MultiEncoder) string {
 		if err == nil {
 			s = s2
 		}
+	} else {
```

Comment:
We probably don't need to so this on Windows now do we? Either we've made the UNC already which is by definition absolute or the path is Abs and clean.